model/lifecycle.go

```diff
@@ -778,8 +778,11 @@ func createTasksForBuild(creationInfo TaskCreationInfo) (task.Tasks, error) {
 			return nil, errors.Wrapf(err, "creating task '%s'", id)
 		}
 
-		// set Tags based on the spec
-		newTask.Tags = creationInfo.Project.GetSpecForTask(t.Name).Tags
+		projectTask := creationInfo.Project.FindProjectTask(t.Name)
+		if projectTask != nil {
+			newTask.Tags = projectTask.Tags
+		}
+		newTask.Tags = creationInfo.Project.FindProjectTask(t.Name).Tags
```

Comment:
Seems like L785 is supposed to be removed?

Reply:
yeah sorry @hadjri it was supposed to be removed which removes the need for the empty project task to prevent panics